docs/outposts.md

```diff
@@ -0,0 +1,51 @@
+# Using EBS CSI Driver on AWS Outposts
+
+## Overview
+
+When using the EBS CSI Driver on AWS Outposts, there are several important considerations to be aware of.
+
+### IMDS Metadata Access
+The EBS CSI driver requires access to Instance Metadata Service (IMDS) to function properly on Outposts. Ensure that:
+- IMDS is not disabled on your EC2 instances
+- If using IMDSv2 ensure the hop limit is set to 2 
```

Comment:
Remove "If using IMDSv2" -> it's confusing and I don't believe the SDK even supports v1 anymore.